src/Symfony/Component/Security/Http/Event/LogoutUserChangeEvent.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Http\Event;
+
+use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
+use Symfony\Component\Security\Core\User\UserInterface;
+use Symfony\Contracts\EventDispatcher\Event;
+
+/**
+ * @author Hamza Amrouche <hamza.simperfit@gmail.com>
+ */
+class LogoutUserChangeEvent extends Event
```

Comment:
Not too sure about the name. What about "LogoutOnUserChangedEvent"? It captures the actual scenario a bit better

Reply:
What about `LogoutOnChangeEvent` due to `security.logout_on_change`?

to map the `SwitchUserEvent` => `security.switch_user`